realsense_ros_camera/src/base_realsense_node.cpp

```diff
@@ -917,32 +925,179 @@ void BaseRealSenseNode::publish_static_tf(const ros::Time& t,
     msg.header.stamp = t;
     msg.header.frame_id = from;
     msg.child_frame_id = to;
-    msg.transform.translation.x = trans.z;
-    msg.transform.translation.y = -trans.x;
-    msg.transform.translation.z = -trans.y;
+    msg.transform.translation.x = -trans.z;
```

Comment:
Is there a reason we changed the signs for
 these translations?

Reply:
It appears a bug for the driver that the translation is reversed. https://github.com/intel-ros/realsense/issues/327
I had the same issue when testing.

Reply:
Add link comment to code to explain.